### PR TITLE
Quarto leaderboard

### PR DESCRIPTION
**Pull request by rlrs**
Hi guys, I heard you needed a leaderboard website and I wanted to try out Quarto 😉 

It's very simple and extensible, plus it makes static pages that we can just put on Github Pages or anywhere we'd like.
If you think it's ugly we can choose a new theme, or CSS it up. I spent like 20 minutes on it.

### CODE REVIEW

**Review by saattrupdan — COMMENTED** [review]
It seems like quarto needs to be installed separately, and not as a Python package? In that case, we probably need to use Docker or something like it to set up the handling of the leaderboards, no?

Also, we would need separate scripts that populate the JSON files from the database, but that could be a separate PR.

**Comment by rlrs** [comment]
Quarto is only needed to generate the static pages though - data is loaded dynamically with some Javascript on page load.
Oh, and I just chose json randomly - we can integrate with something else probably. Or write a script that pulls data into json, sure.

File: .python-version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.10
```
Comment (saattrupdan) [inline .python-version:1]: Is this needed, or can this be pulled from pyproject.toml instead?

**Review by saattrupdan — CHANGES_REQUESTED** [review]
Just a small question remains from my side. Otherwise its great 😊

We'd need some kind of Docker script to generate the leaderboards then, I think. Just installing quarto, pulling data from the database and generating the static files. But again, that's a separate PR.

**Comment by rlrs** [comment]
> Just a small question remains from my side. Otherwise its great 😊
> 
> We'd need some kind of Docker script to generate the leaderboards then, I think. Just installing quarto, pulling data from the database and generating the static files. But again, that's a separate PR.

Yeah, I'm thinking about the flow a little bit. I also don't think the leaderboard site should be located in this repo, so I'm closing this and moving it elsewhere.

**Pull request closed without merging**